src/components/map/RealtimeLayersService.js

```diff
@@ -10,14 +10,16 @@ goog.require('ga_map_service');
 
   module.provider('gaRealtimeLayersManager', function() {
     this.$get = function($rootScope, $http, $timeout, gaLayerFilters,
-        gaMapUtils, gaUrlUtils) {
+        gaMapUtils, gaUrlUtils, gaLayers) {
 
       var timers = [];
       var realTimeLayersId = [];
       var geojsonFormat = new ol.format.GeoJSON();
 
-      function setLayerSource(layer) {
-        var fullUrl = gaUrlUtils.proxifyUrl(layer.geojsonUrl);
+      function setLayerSource(layer, isNewSource) {
+        var fullUrl = isNewSource ?
+            gaLayers.getLayerProperty(layer.bodId, 'geojsonUrl') :
```

Comment:
`proxifyUrl`  was used because there was no CORS headers so probably you still  need it, unless somthing change server side.